android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -243,6 +389,19 @@ public object WorkerBinder {
       }
     }
   }
+
+  private fun getDispatcherFromMigration(
```

Comment:
The set of APIs around the migration feels like we're leaking to much internal business logic into the public APis of Ribs. We should generalize this more. Perhaps just one override where we can set a static overriding dispatcher.

Reply:
I agree but also I'm a bit concerned that providing an easy static dispatcher override could lead to unwanted dispatchers change at any point by any consumers.

Let me think about this a bit more

Reply:
@tyvsmith Decided to keep this migration in internal monorepo. This PR will just focus on deprecating existing API and add equivalent new methods for the default binding happening off the main thread